phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexTool.java

```diff
@@ -85,7 +102,7 @@
     private static final Option DATA_TABLE_OPTION = new Option("dt", "data-table", true,
             "Data table name (mandatory)");
     private static final Option INDEX_TABLE_OPTION = new Option("it", "index-table", true,
-            "Index table name(mandatory)");
+            "Index table name(not required in case of partial rebuilding)");
```

Comment:
Better to have an argument for partial rebuilding than go by rebuild if we don't mention index table.